src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -20,52 +20,53 @@ export class TransactionDetailsSummaryItem extends React.Component<
   TransactionDetailsSummaryItemProps
 > {
   render() {
-    const {
-      offerOverride,
-      order: {
-        itemsTotal,
-        mode,
-        totalListPrice,
-        shippingTotal,
-        shippingTotalCents,
-        taxTotal,
-        taxTotalCents,
-        buyerTotal,
-      },
-      ...others
-    } = this.props
-
-    console.log("FUXKKC", this.props.order)
-
+    const { offerOverride, order, ...others } = this.props
     return (
       <StackableBorderBox flexDirection="column" {...others}>
-        {mode === "OFFER" ? (
-          <>
-            {/* TODO: Seller's offer / Your offer (/ Buyer's offer? Will sellers see this component?) */}
-            <Entry label="Your offer" value={offerOverride || itemsTotal} />
-            {Boolean(itemsTotal) && (
-              <SecondaryEntry label="List price" value={totalListPrice} />
-            )}
+        {/* TODO: Seller's offer / Your offer (/ Buyer's offer? Will sellers see this component?) */}
+        {this.priceDisplayComponent(order, offerOverride)}
+        <Spacer mb={2} />
+        <Entry label="Shipping" value={this.shippingDisplayAmount(order)} />
 
-            <Spacer mb={2} />
-          </>
-        ) : (
-          <Entry label="Price" value={itemsTotal} />
-        )}
-        <Entry
-          label="Shipping"
-          value={this.formattedAmount(shippingTotal, shippingTotalCents) || "—"}
-        />
-        <Entry
-          label="Tax"
-          value={this.formattedAmount(taxTotal, taxTotalCents) || "—"}
-        />
+        <Entry label="Tax" value={this.taxDisplayAmount(order)} />
         <Spacer mb={2} />
-        <Entry label="Total" value={buyerTotal} final />
+        <Entry label="Total" value={order.buyerTotal} final />
       </StackableBorderBox>
     )
   }
 
+  private shippingDisplayAmount = order => {
+    return order.mode === "BUY"
+      ? this.formattedAmount(order.shippingTotal, order.shippingTotalCents) ||
+          "—"
+      : this.formattedAmount(
+          order.myLastOffer.shippingTotal,
+          order.myLastOffer.shippingTotalCents
+        ) || "—"
+  }
+  private taxDisplayAmount = order => {
+    return order.mode === "BUY"
+      ? this.formattedAmount(order.taxTotal, order.taxTotalCents) || "—"
+      : this.formattedAmount(
+          order.myLastOffer.taxTotal,
+          order.myLastOffer.taxTotalCents
+        ) || "—"
+  }
+
+  private priceDisplayComponent = (order, offerOverride) => {
```

Comment:
#superminor 

A convention that I've seen in react codebases is that component methods that return UI are prefixed with `render`, so this would be `renderPriceEntry` or similar